terraform/cloud-platform-components/prometheus.tf

```diff
@@ -8,9 +8,11 @@ resource "kubernetes_secret" "grafana_secret" {
   }
 
   data {
-    GF_AUTH_GITHUB_CLIENT_ID     = "${ var.github_client_id }"
-    GF_AUTH_GITHUB_CLIENT_SECRET = "${ var.github_client_secret }"
-    GF_SECURITY_SECRET_KEY       = "${ var.github_secret_key }"
+    GF_AUTH_GENERIC_OAUTH_CLIENT_ID     = "${ var.auth0_client_id }"
```

Comment:
These variables should be picked up from the `cloud-platform` [state](https://github.com/ministryofjustice/cloud-platform-infrastructure/blob/live-0/terraform/cloud-platform/outputs.tf#L53-L71). There is an auth0 client created there to be used for components.

Having them as variables in this terraform configuration means we need to manually create the app for every cluster and also manually pass them in every time we need to run terraform, which is something we should avoid.